cli/args/flags_net.rs

```diff
@@ -32,7 +32,7 @@ pub fn validator(host_and_port: &str) -> Result<String, String> {
   if Url::parse(&format!("internal://{host_and_port}")).is_ok()
     || host_and_port.parse::<IpAddr>().is_ok()
     || host_and_port.parse::<BarePort>().is_ok()
-    || NetDescriptor::parse(host_and_port).is_ok()
+    || NetDescriptor::parse_for_list(host_and_port, true).is_ok()
```

Comment:
I see from a resolved comment that we're going to change this to an enum? Consider that or have a `parse_for_list_with_wildcards` method.

Reply:
+1 here - ideally we'd find a way to ban `bool` in permission code :D everything should be an enum here with descriptive name